content/security/committers.md

```diff
@@ -1,110 +1,118 @@
 
 1. The person discovering the issue, the _reporter_, reports the
 vulnerability privately to `security@project.apache.org` or to
 `security@apache.org`.
 
-1. Messages that do not relate to reporting or managing an
-undisclosed security vulnerability in Apache software are ignored and no
-further action is required.
+2. The Security team ignores any message to this inbox that does not relate to reporting or managing an
+undisclosed security vulnerability in Apache software.
 
-1. If the report comes to `security@apache.org`, the security team forwards
+3. If the report comes to `security@apache.org`, the security team forwards
 it to the project's security list or, if the project does not
 have a security list, to the project's private (PMC) mailing list.
-The security team responds to the original reporter that this has
-been done.
+The Security Team responds to the original reporter that they have done this.
+
+### Acknowledge
 
-1. The project team sends an e-mail to the original reporter to acknowledge the report, with a copy to `security@project.apache.org` if it exists, or
+4. The project team sends an e-mail to the original reporter to acknowledge the report, with a copy to `security@project.apache.org` if it exists, or to
 `security@apache.org`.
 
-1. The project team investigates the report and either rejects or accepts
+5. The project team investigates the report and either rejects or accepts
 it.
 
-1. If the project team rejects the report, the team writes to the reporter to
+6. If the project team **rejects** the report, the team writes to the reporter to
 explain why, with a copy to `security@project.apache.org` if it exists, or to
 `security@apache.org`.
 
-1. If the project team accepts the report, the team writes to the reporter to let them
+7. If the project team **accepts** the report, the team writes to the reporter to let them
 know that they have accepted the report and that they are working on a fix.
 
-1. The project team requests a CVE (<a href="https://cve.mitre.org/" target="_blank">Common Vulnerabilites and Exposures</a>) name from the internal portal, `https://cveprocess.apache.org`.  Alternatively by
+8. The project team requests a CVE (<a href="https://cve.mitre.org/" target="_blank">Common Vulnerabilites and Exposures</a>) ID from the internal portal, `https://cveprocess.apache.org`; or by
 sending an e-mail with the subject "CVE request for..." to `security@apache.org`, providing a
 short (one-line) description of the vulnerability. `security@apache.org` can
-help determine if a report requires multiple CVEs or if multiple reports
-should be merged under a single CVE.
+help determine if a report requires multiple CVE IDs or if multiple reports
+should be merged under a single ID.
 
-1. The ASF security team will allocate a CVE and send a link to the
-internal portal where the project team can enter details of the
+9. The ASF security team allocates a CVE ID and sends to the project team a link to the
```

Comment:
The new language here feels backwards.

"and sends a link to the internal portal to the project team. The project team adds information about the vulnerability using the portal. "  (is that second item a different bullet point?

Reply:
I agree with changing the text. I felt the language was passive/oddly ordered: 

"sends to the project team a link"  vs  "sends a link to the project team". 

After that I was trying to then get the language to fit, but that's really my feedback.  Send X to Y rather than Send to X a Y.

Reply:
Neither version is passive voice. The problem is the "...to the....to the..." construction of the sentence. Do you lead with what you are sending, or the group to whom you are sending it?